app/editor/src/features/content/list-view/ContentListView.tsx

```diff
@@ -50,6 +52,9 @@ const ContentListView: React.FC = () => {
   const [contentType, setContentType] = React.useState(formType ?? ContentTypeName.AudioVideo);
   const [isLoading, setIsLoading] = React.useState(false);
 
+  const [results, setResults] = React.useState<IContentSearchResult[]>([]);
+  const [{ actions }] = useLookup();
```

Comment:
None state hooks belong at the top by convention and organization.

Reply:
Updated, thanks